### PR TITLE
9063 - Add SEO amends for lbbt and ltt calculators

### PR DESCRIPTION
**Pull request by tomas-stefano**
[TP card](https://moneyadviceservice.tpondemand.com/entity/9063-seo-amends-for-land-and-buildings)

This PR addresses the following tasks:

* Add a heading description for improving the SEO value.
* Add meta tags for each calculator
* Changing links on the link to other tools section on each calculator

### CODE REVIEW

File: spec/helpers/application_helper_spec.rb [hunk 1]
```diff
@@ -2,21 +2,53 @@
...
+          expect(helper.calculator_name).to eq('land_and_buildings_transaction_tax')
+        end
+      end
+
+      context 'when land transaction taxes controller' do
+        let(:controller_name) do
+          'land_transaction_taxes'
+        end
+
+        it "returns 'land_transaction_tax'" do
```
Comment (willhall88) [inline spec/helpers/application_helper_spec.rb]: this is minor, but what is the agreed standard for speech marks within speech marks? 
here you have `"returns 'land_transaction_tax'"` above you have `'returns "affordability"'`

**Comment by margOnline** [comment]
👍

File: spec/helpers/application_helper_spec.rb (hunk 1 above)
Comment (tomas-stefano) [reply spec/helpers/application_helper_spec.rb]: We don't have one but IMHO it should be `'returns "affordability"'. I change acoordingly to be consistent.

Comment (willhall88) [reply spec/helpers/application_helper_spec.rb]: agreed! thanks

**Pull request merged**